.gitignore

```diff
@@ -4,3 +4,4 @@
 *.apk
 *.tar.gz
 .vscode
+.idea
```

Comment:
Do not add an unrelated change.

Reply:
I wounder, add it in separate PR or not add at all? This is JetBrains IDEs settings

Reply:
We can add it in a separate repository. Or, you can add your own global gitignore setting